### PR TITLE
DVISA-1673: move fork changes to cornerstone3d repo

### PR DESCRIPTION
**Pull request by felixschuller**
# What does this Merge Request do?

Moves all changes from our old fork to the new one.

## Related Jira issues

http://jira.dedalus.com/browse/DVISA-1673

## Review instructions

Check if all our introduced changes are applied to the new loader as well. See: https://github.com/cornerstonejs/cornerstoneWADOImageLoader/compare/master...DedalusDIIT:cornerstoneWADOImageLoader:master

### CODE REVIEW

File: yarn.lock
Comment (aminahasimbegovic) [inline yarn.lock:1]: why was this file changed?

**Review by aminahasimbegovic — APPROVED** [review]
Nice!! :)

File: yarn.lock
Comment (muellnerlisa) [reply yarn.lock:1]: this changed after running npm i in the project, maybe its because we are not using yarn and npm instead
